src/iris-mobile/src/APIServer/oncallSync.py

```diff
@@ -0,0 +1,46 @@
+import requests
```

Comment:
Does iris-mobile share code with iris-api? If it doesn't, its codebase should really go into a separate subfolder under the repo root. Such as /iris-mobile